components/actions/actionsSection.tsx

```diff
@@ -319,7 +331,18 @@ export default function PageActionsSection(props: {
             </p>
           </div>
         )}
-        {numActiveActions > 20 && (
+        {actionsExceedMaxTokens && (
+          <div
+            className="fixed bottom-0 inset-x-0 md:mx-10 lg:mx-auto max-w-7xl flex flex-row gap-x-2 bg-red-900 border-l-4 border-red-500 text-white px-4 py-6"
+            role="alert"
+          >
+            <p className="font-bold">Error:</p>
+            <p>
+              {`You have ${numActiveActions} actions enabled. The language model cannot process this many. We recommend enabling around 20 actions.`}
```

Comment:
I see the reasoning for putting it in terms of 'actions', but suppose there's 1 action which is enormously long and the others are normal-sized. They probably ought to know this.

My alternative suggestions are (pick one):
1. Point them to the 'view system prompt' modal so they can see how long each action is
2. Calculate a rough token estimate for each action and tell them the 5 highest token actions